coc/index.html.j2

```diff
@@ -0,0 +1,66 @@
+{% set title = "Code of Conduct" %}
+{% extends "base.html.j2" %}
+
+{% block head %}
+  <style>
+    .person {
+      border-radius: var(--border-radius);
+      border: var(--border-width) solid var(--mute-border-color);
+      max-width: 200px;
+      overflow: hidden;
+    }
+
+    .person > figcaption {
+      font-family: var(--header-font);
+      letter-spacing: 0.03125em;
+      padding-inline: 1rem;
+      text-align: left;
+      text-transform: uppercase;
+    }
+  </style>
+{% endblock %}
+
+{% block main %}
+  <h1>Code of Conduct</h1>
+
+  <p class="lead">
+
+PGConf.dev, as a PostgreSQL Community Conference, has adopted the <a href="https://www.postgresql.org/about/policies/coc/">PostgreSQL Code Of Conduct policy</a>. Please be sure to review that policy.
```

Comment:
We're saying we've adapted one code of Conduct, but then proceed to express another one. While the goal is obviously the same, the details are not and we should be clear about which code of Conduct actually applies.

Reply:
Is there a specific change you propose ?
Are you saying that we shouldn't reference the community COC but embed the content here?

Reply:
We already embed the content of *our* CoC, that's what the whole page is about, no? We're specifically not adopting the central CoC, we have our own, and this patch adds the content of our own?

Reply:
I've removed the first paragraph that references the community coc page